install/kubernetes/ansible/istio/tasks/delete_resources.yml

```diff
@@ -12,7 +12,7 @@
 
 - name: Delete Kiali info
   shell: |
-    {{ cmd_path }} delete all,secrets,sa,templates,configmaps,deployments,clusterroles,clusterrolebindings,routerules --selector=app=kiali --selector=version=master --ignore-not-found=true -n {{ istio_namespace }}
+    {{ cmd_path }} delete all,secrets,sa,templates,configmaps,deployments,clusterroles,clusterrolebindings,virtualservices,destinationrules  --selector=app=kiali --selector=version=master --ignore-not-found=true -n {{ istio_namespace }}
```

Comment:
You will not need this if https://github.com/istio/istio/pull/7741 got merged?

Reply:
Right!

I think #7736 is a requirement for #7741

Reply:
Can you rebase as #7741 got merged.